wcivf/apps/elections/templates/elections/includes/_polling_place.html

```diff
@@ -46,7 +45,9 @@ <h3>{{ voter_id_required.0.1.title }}</h3>
     {% else %}
       {% now "j F Y" as current_day %}
       {% if elections_by_date and elections_by_date.0.grouper|date:"j F Y" == current_day %}
-      <strong>Polling stations are open from 7am till 10pm today.</strong>
+        {% with today_election=elections_by_date.0.list.0.election %}
```

Comment:
Untested, but I think `postelections` still exists in this context, meaning you should be able to do `today_election=postelections.0.election` - this might make it a little easier to understand?